route-api/main.ts

```diff
@@ -1,5 +1,8 @@
 import mqtt from "mqtt";
 import _mariadb from "mariadb";
+import { Application } from "oak";
+
+const hereTransitAPIKey = Deno.env.get("HERETRANSITAPI");
```

Comment:
as an external constant convention is uppercase + snakecase

```suggestion
const HERE_TRANSIT_API_KEY = Deno.env.get("HERE_TRANSIT_API");
```